test/multiverse/suites/redis/redis_instrumentation_test.rb

```diff
@@ -380,5 +382,13 @@ def client
         :_client
       end
     end
+
+    def redis_host
+      docker? ? 'redis' : NewRelic::Agent::Hostname.get
+    end
+
+    def either_hostname
+      [NewRelic::Agent::Hostnam, 'redis']
```

Comment:
Should this be "Hostname"? (I think this is why some of the CI suites failed)

Reply:
Oof! Fixed with f2c1039, thank you.